planner_cspace/test/CMakeLists.txt

```diff
@@ -57,6 +57,11 @@ add_rostest(test/navigation_rostest.test
   DEPENDENCIES test_navigate
 )
 
+add_rostest(test/navigation_rostest.test
```

Comment:
As the executed file is same as the other tests, coverage profile seems be overwritten by the last run of the executable.
I suppose it requires to set unique `GCOV_PREFIX` and `GCOV_PREFIX_STRIP` environment variable for each rostest run to keep all coverage profiles.
https://gcc.gnu.org/onlinedocs/gcc-5.2.0/gcc/Cross-profiling.html

Reply:
https://github.com/at-wat/neonavigation/issues/519